terra/VteObject.py

```diff
@@ -55,15 +54,22 @@
     "(?:news:|man:|info:)[[:alnum:]\\Q^_{|}~!\"#$%&'()*+,./;:=?`\\E]+"]
 
 class VteObjectContainer(Gtk.HBox):
-    def __init__(self, parent, bare=False, progname=ConfigManager.get_conf('general', 'start_shell_program'), pwd=None):
+    def __init__(self, parent, bare=False, progname=None, pwd=None):
```

Comment:
Yes, for readability mainly.  But I want to change the code so that you don't need so many dependencies upon `from terra.xyz imports ClassXYZ`.